backend/server/package.json

```diff
@@ -8,6 +8,7 @@
     "build": "tsoa spec-and-routes && tsc",
     "start": "node build/src/index.js",
     "dev": "concurrently \"nodemon\" \"nodemon -x tsoa spec-and-routes\"",
+    "local": "tsx watch src/index.ts",
```

Comment:
Isn't this just a dupe `dev` script without nodemon but with tsx -watch instead? Is there any benefit to using this over the dev script?

Reply:
I couldnt run the dev script when I was working. I didn't want to spend a lot of time on it, so I just used local instead. We can remove it later.

Reply:
Did you run `npm install`? :smile:

Reply:
resolved. dont know how/why it wasn't working the first time. I removed the local cmd.